config/obstacles.json

```diff
@@ -3,1233 +3,1206 @@
+		"graphics" : "ObBhL06.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 27,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [0],
-		"width" : 261,
-		"height" : 129,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [1],
+		"offsetGraphicsInX" : 261,
+		"offsetGraphicsInY" : 129,
 		"blockedTiles" :  [27, 28, 43, 44, 60, 61, 76, 77, 93, 94, 109, 110, 126, 127, 142, 143, 159],
-		"defname" : "ObBhL14.pcx"
+		"graphics" : "ObBhL14.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 28,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [0],
-		"width" : 180,
-		"height" : 154,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [1],
+		"offsetGraphicsInX" : 180,
+		"offsetGraphicsInY" : 154,
 		"blockedTiles" :  [22, 38, 39, 40, 44, 45, 46, 55, 56, 57, 62, 63, 123, 124, 125, 130, 131, 140, 141, 146, 147, 148],
-		"defname" : "ObBhL16.pcx"
+		"graphics" : "ObBhL16.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 29,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [5],
-		"width" : 304,
-		"height" : 264,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [19],
+		"offsetGraphicsInX" : 304,
+		"offsetGraphicsInY" : 264,
 		"blockedTiles" :  [76, 77, 92, 93, 94, 95, 109, 110, 111],
-		"defname" : "ObCFL00.pcx"
+		"graphics" : "ObCFL00.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 30,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [6],
-		"width" : 256,
-		"height" : 257,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [17],
+		"offsetGraphicsInX" : 256,
+		"offsetGraphicsInY" : 257,
 		"blockedTiles" :  [76, 77, 78, 92, 93, 94, 107, 108, 109],
-		"defname" : "ObLPL00.pcx"
+		"graphics" : "ObLPL00.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 31,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [7],
-		"width" : 257,
-		"height" : 255,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [14],
+		"offsetGraphicsInX" : 257,
+		"offsetGraphicsInY" : 255,
 		"blockedTiles" :  [76, 77, 91, 92, 93, 94, 95, 108, 109, 110, 111],
-		"defname" : "ObFFL00.pcx"
+		"graphics" : "ObFFL00.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 32,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [8],
-		"width" : 277,
-		"height" : 218,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [15],
+		"offsetGraphicsInX" : 277,
+		"offsetGraphicsInY" : 218,
 		"blockedTiles" :  [60, 61, 75, 76, 77, 91, 92, 93, 94, 95],
-		"defname" : "ObRLL00.pcx"
+		"graphics" : "ObRLL00.pcx",
+		"canBeRemoved" : false,
 	},
 	{
-		"id" : 33,
-		"allowedTerrain" : [],
-		"specialBattlefields" : [9],
-		"width" : 300,
-		"height" : 214,
+		"type" : "static",
+		"position" : 0,
+		"battlefieldSurface" : [16],
+		"offsetGraphicsInX" : 300,
+		"offsetGraphicsInY" : 214,
 		"blockedTiles" :  [59, 60, 74, 75, 76, 93, 94, 95, 111, 112],
-		"defname" : "ObMCL00.pcx"
+		"graphics" : "ObMCL00.pcx",
+		"canBeRemoved" : false,
+	}
+],
+"InherentObstacles" : [

```

Comment:
use camelCase for JSON field names

Reply:
I suppose you mean lower camel case, since I used upper camel case.